src/main/java/mate/academy/service/impl/CinemaHallService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.service.impl;
```

Comment:
The package name `mate.academy.service.impl` suggests that this file contains an implementation, but it actually defines an interface. Consider renaming the package to `mate.academy.service` to better reflect its contents.